test/e2e_scheduler/waitforfirstconsumer.go

```diff
@@ -0,0 +1,90 @@
+package e2escheduler
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/loft-sh/vcluster/test/framework"
+	"github.com/onsi/ginkgo"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var _ = ginkgo.Describe("Schedule a Statefulset with WaitForFirstConsumer PVCs", func() {
+	// the default kind storageclass is WaitForFirtConsumer
```

Comment:
> // the default kind storageclass is WaitForFirtConsumer

I don't think we should rely on this^. For example, in minikube, the default StorageClass has `volumeBindingMode: Immediate`, so if someone runs the tests locally, or if we switch the k8s used in the CI, or the KinD default changes - the test would always pass(with potential false positive).

Reply:
Makes sense! I can fetch and make a copy of the default storageclass that has the correct binding mode.

Reply:
Yeah, that would be nice. But this is not a blocker, so if you are busy with other things now, we can follow up in a separate PR.

Reply:
Fixed!